api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -327,6 +328,10 @@ message ExternalProcessor {
   // :ref:`mode_override <envoy_v3_api_field_service.ext_proc.v3.ProcessingResponse.mode_override>` is allowed by
   // the ``allowed_override_modes`` allow-list below.
   repeated ProcessingMode allowed_override_modes = 22;
+
+  // Decorator to introduce custom logic that runs after a message received from
+  // the External Processor is processed.
+  config.core.v3.TypedExtensionConfig on_processing_response = 23;
```

Comment:
Current API LGTM, other than the extensions category.
Let's see if we can add a simple extension.

Reply:
Added extensions category.